File: 30 days of code( python)/Day 20.py
#!/bin/python

import sys


n = int(raw_input().strip())
a = map(int,raw_input().strip().split(' '))

def bubble_sort(array):
    num_of_swaps = 0
    for i in range(len(array)):
        for index in range(len(array)):
            try:
                if (array[index] > array[index + 1]):
                    temp = array[index]
                    array[index] = array[index + 1]
                    array[index + 1] = temp
                    num_of_swaps = num_of_swaps + 1
                
            except:
                continue
            
            
    return [num_of_swaps, array[0], array[-1]]
 
result = bubble_sort(a)
print "Array is sorted in", result[0], "swaps."
print "First Element:", result[1]
print "Last Element:", result[2]

File: 30daysOFcode_TypeScript/day 20.ts
function main(): void {
    let n: number = parseInt(readLine());
    let a: Array<number> = readLine().split(' ');
    a = a.map(Number);

    let noOfSwaps: number = 0;

    for (let i: number = 0; i < n - 1; i++) {
        for (let j: number = i + 1; j < n; j++) {
            if (a[i] > a[j]) {
                let temp: number = a[i];
                a[i] = a[j];
                a[j] = temp;
                noOfSwaps++;
            }
        }
    }

    console.log('Array is sorted in ' + noOfSwaps + ' swaps.');
    console.log('First Element: ' + a[0]);
    console.log('Last Element: ' + a[n - 1]);
}
